helpdesk/email.py

```diff
@@ -468,11 +465,10 @@ def is_autoreply(message):
 def create_ticket_cc(ticket, cc_list):
     if not cc_list:
         return []
-
     # Local import to deal with non-defined / circular reference problem
-    from helpdesk.views.staff import subscribe_to_ticket_updates, User
 
     new_ticket_ccs = []
+    from helpdesk.views.staff import subscribe_to_ticket_updates, User
```

Comment:
I can see that foreign users may want the default name of the file to be in their own language so I guess that is why it was set up as a translation originally and removing it would change existing behaviour.
https://github.com/django-helpdesk/django-helpdesk/blob/main/helpdesk/email.py#L784

Reply:
Alright, now I understand